packages/keycloak-user-management/src/components/UserList/index.tsx

```diff
@@ -82,10 +82,8 @@ const UserList = (props: Props): JSX.Element => {
       serve
         .list()
         .then((res: KeycloakUser[]) => {
-          if (isLoading) {
-            setIsLoading(false);
-            fetchKeycloakUsersCreator(res);
-          }
+          setIsLoading(false);
+          fetchKeycloakUsersCreator(res);
```

Comment:
Why are we doing this

Reply:
It was a duplicate condition
the condition was already applied here. 
https://github.com/OpenSRP/web/blob/master/packages/keycloak-user-management/src/components/UserList/index.tsx#L80